AllHitAnalyze_New.py

```
@@ -1,162 +1,162 @@
+		#   [±5ms, ±10ms, ±20ms, ±45ms, Blue Exact, Great, Right, Miss]
+		self.Accurate_Sync_Rate:list[int] = [0] * 8;
+		# init hit map: [-149, -0], [+0, +250]
+		# self.hitMapPositive:list[int] = [0]*150;
+		# self.hitMapNegative:list[int] = [0]*251;
+		# init chart axis
+		self.xAxis:list[int] = [i for i in range(-150, 251, 1)];
+		self.yAxis:list[int] = [0] * 401 # self.hitMapPositive + self.hitMapNegative;
+		self.LoadHitMap(0);
+		# All Hit PDF: Normal Distribution
+		# avg:Average , var:Variance ,std: Standard Deviation
+		self.avg:float = sum([num[0] * num[1] / self.summation for num in zip(self.xAxis, self.yAxis)]);
```

Ruff (0.14.8)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)